1.hello-world/README.md

```diff
@@ -135,7 +135,7 @@ You're up and running! Both Dapr and your app logs will appear here.
 ```
 > **Note**: the `--app-port` (the port the app runs on) is configurable. Our Node app happens to run on port 3000, but we could configure it to run on any other port. Also note that the Dapr `--port` parameter is optional, and if not supplied, a random available port is used.
 
-The `dapr run` command looks for a `components` directory which holds yaml definition files for components Dapr will be using at runtime. When running locally, if the directory is not found it is created with yaml files which provide default definitions for a local development environment (learn more about this flow [here](https://github.com/dapr/docs/blob/master/walkthroughs/darprun.md)). Review the `statestore.yaml` file in the `components` directory:
+The `dapr run` command looks for the default components directory under `~/.dapr/components` which holds yaml definition files for components Dapr will be using at runtime. When running locally, the yaml files which provide default definitions for a local development environment are placed in this default components directory (learn more about this flow [here](https://github.com/dapr/docs/blob/master/walkthroughs/darprun.md)). Review the `statestore.yaml` file in the `components` directory:
```

Comment:
This is a good description of default usage when the components dir is not user-specified.

Can you add an example to a `dapr run` using `--components-path` too?  It should be clear on whether yamls will be put there or not by the cli, or whether it's expected that the user have pre-populated it.

Reply:
I thought about it but did not want to add it in the introductory hello-world sample